src/PetProjects.Transactions.Contracts/Topics/V1/Events/TransactionEvents.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PetProjects.Transactions.Contracts.Topics.V1.Events
```

Comment:
Why PetProjects.Transactions.Contracts ? Are we supposed to use the .Transactions namespace for everything related to the MicroTransactions pet project? Because if so we need to change the namespace of the log-aggregator and the APIs

Reply:
The idea was having this contracts related to Transactions API. The other should/could have their own contracts.

Reply:
The general idea is to have contracts(topics, messages) per entity.

Reply:
So would we have a different repository for another entity? Or a different .sln file for a different entity in this repository? The sln should have a namespace that is shared by every .csproj/.nuget of that sln. Something like PetProjects.Contracts.[Entity].Topics, PetProjects.Contracts.[Entity].Commands, PetProjects.Contracts.[Entity].Events.
With this namespace structure we can decide whether we want a csproj per entity or a csproj for all entities with a folder for Topics, another for Commands and another for Events. 

An alternative would be PetProjects.Contracts.Topics.[Entity], PetProjects.Contracts.Commands.[Entity] and PetProjects.Contracts.Events.[Entity]. With these namespaces we could have one csproj for everything or one csproj for commands, one for events and one for topics (each one with different folders for each entity).

I prefer the latter alternative personally.

Reply:
My take would be. One for commands and one for events. Since the topics must be together with the messages. I think a csproj just for topics is not necessary.

Reply:
This means each entity will have 2 .csproj (commands and events).

Reply:
I think that having 2 csproj per entity is a nightmare to scale. If we have 5 entities in our system we will have 10 projects/nugets in a single sln... And 5 is a relatively small number.

Reply:
If we separate Commands from Events in different projects then where do we put the Topics if we don't have a project for these?

Reply:
The topic should stay alongside its messages.
CommandsTopic with Command  Messages.
EventsTopic with Events Messages.

As we discussed, we will have a project for each API. With its corresponding events and commands.